src/common/utxobased/engine/makeUtxoEngine.ts

```diff
@@ -132,6 +140,209 @@ export async function makeUtxoEngine(
   })
 
   const engine: EdgeCurrencyEngine = {
+    async accelerate(edgeTx: EdgeTransaction): Promise<EdgeTransaction | null> {
+      // Get the replaced transaction rom the processor:
+      const replacedTxid = edgeTx.txid
+      const [replacedTx] = await processor.fetchTransactions({
+        txId: replacedTxid
+      })
+
+      // Transaction checks:
+      // The transaction must be found and not confirmed or dropped.
+      if (replacedTx == null) return null
+      if (replacedTx.blockHeight !== 0) return null
+
+      // Double the fee used for the RBF transaction:
+      const vBytes = transactionSizeFromHex(replacedTx.hex)
+      const newFeeRate = Math.round(parseInt(replacedTx.fees) / vBytes) * 2
```

Comment:
Put the `* 2` inside the `Math.round`, so we do the doubling in float-land instead of integer-land.

Rounding creates ±0.5 of "random noise" in the output values, but the `* 2` doubles this noise to ±1. If you perform the `* 2` before rounding, though, you only get the ±0.5 of noise needed to hit an integer, and nothing more.